src/VirtoCommerce.SimpleExportImportModule.Data/Services/CsvPagedPriceDataSource.cs

```diff
@@ -41,6 +41,33 @@ public CsvPagedPriceDataSource(IProductSearchService productSearchService, Strea
 
         public int PageSize { get; }
 
+        public string GetHeaderRaw()
+        {
+            string result = null;
+
+            var streamPosition = _stream.Position;
+            _stream.Seek(0, SeekOrigin.Begin);
```

Comment:
If GetHeaderRaw would be called between FetchAsync calls, does streams would be rewinded?